app/models/manageiq/providers/azure/inventory/parser/network_manager.rb

```diff
@@ -86,10 +86,33 @@ def cloud_subnets(persister_cloud_networks, cloud_network)
         :cidr              => subnet.properties.address_prefix,
         :cloud_network     => persister_cloud_networks,
         :availability_zone => persister.availability_zones.lazy_find('default'),
+        :network_router    => persister.network_routers.lazy_find(subnet.properties.try(:route_table).try(:id))
       )
     end
   end
 
+  def network_routers
```

Comment:
you need to call this method above if ```def parse``` otherwise it's not included

Reply:
Missed that, thanks.